eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -115,7 +114,6 @@ public function storeImageReference($uri, $fieldId)
      * Return an XML content stored for the given $fieldIds.
      *
      * @param int $versionNo
-     * @param array $fieldIds
```

Comment:
Why remove this?

Reply:
> Why remove this?

Redundant, available in the method signature.